nixos/modules/services/web-apps/mattermost.nix

```diff
@@ -207,8 +206,7 @@ in
           PermissionsStartOnly = true;
           User = cfg.user;
           Group = cfg.group;
-          ExecStart = "${pkgs.mattermost}/bin/mattermost" +
-            (lib.optionalString (!cfg.mutableConfig) " -c ${database}");
```

Comment:
Why is this changing?

Was it wrong before, or is there a change in the verison upgrade that makes it wrong?

Please also write the answer into the commit message.